README.rst

```diff
@@ -162,3 +162,12 @@ Other Python wrappers for GMT:
 * `gmtpy <https://github.com/emolch/gmtpy>`__ by `Sebastian Heimann <https://github.com/emolch>`__
 * `pygmt <https://github.com/ian-r-rose/pygmt>`__ by `Ian Rose <https://github.com/ian-r-rose>`__
 * `PyGMT <https://github.com/glimmer-cism/PyGMT>`__  by `Magnus Hagdorn <https://github.com/mhagdorn>`__
+
+
+Documentation for other versions
+--------------------------------
+* `Development <https://www.pygmt.org/dev>`__ (reflects the *master* branch on
+  Github)
+* `Latest release <https://www.pygmt.org/latest>`__
```

Comment:
Just a reminder. We might want to switch the HTTP redirect to `latest` instead of `dev` once v0.1.0 is released.